pages/_error.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+function Error({ statusCode }) {
```

Comment:
Must use arrow function -> it's react best practice

const Error = ({ statusCode }) => {

}; <- don't forget to add a semicolon after arrow function

Reply:
done it.